ircd/version.c.SH

```diff
@@ -67,7 +67,8 @@ const char *infotext[] = {
     "The head developer of the u2.10 source tree was:",
     "Bleep        Thomas Helvey            <tomh@inxpress.net>",
     "",
-    "The current maintainors of the u2.10 source tree are:",
+    "The current maintainers of the u2.10 source tree are:",
+    "Entrope      Michael Poole            <mdpoole@troilus.org>",
     "Isomer       Perry Lorier             <perry@coders.net>",
```

Comment:
Isomer resigned as maintainer a long time ago -- we might as well remove him while we're touching this.

Reply:
Done. Modifications tested and force-pushed updated commit.